Tools/autotest/vehicle_test_suite.py

```diff
@@ -8693,6 +8695,11 @@ def sitl_is_running(self):
     def autostart_mavproxy(self):
         return self.use_map
 
+    def set_speedup_parameter(self, speedup):
+        self.set_parameters({
+            "SIM_SPEEDUP": speedup,
```

Comment:
this would slow down tests a bit I think, as it won't be able to set this till parameters are fetched
could we use an additional defaults file?

Reply:
> this would slow down tests a bit I think, as it won't be able to set this till parameters are fetched could we use an additional defaults file?

I did measure and it was within the noise

Reply:
I've reworked this to use a defaults file instead.